decred_wallet/Features/Wallet Setup/RecoverExistingWalletViewController.swift

```diff
@@ -109,57 +126,48 @@ class RecoverExistingWalletViewController: WalletSetupBaseViewController, UITabl
         }
         
         if self.validateSeed().valid {
-            self.activateConfirmButton()
+            self.btnConfirm.backgroundColor = UIColor.appColors.decredGreen
         } else {
-            self.deactivateConfirmButton()
+            self.btnConfirm.backgroundColor = UIColor.appColors.lightGray
         }
     }
     
     func focusSeedWordInput(at tableRowIndex: Int) {
         let tableIndexPath = IndexPath(row: tableRowIndex, section: 0)
-        
         let nextSeedWordCell = self.tableView.cellForRow(at: tableIndexPath) as? RecoveryWalletSeedWordCell
         nextSeedWordCell?.seedWordAutoComplete.becomeFirstResponder()
         
-        self.tableView.scrollToRow(at: tableIndexPath, at: .middle, animated: true)
+        self.tableView.scrollToRow(at: tableIndexPath, at: .bottom, animated: true)
     }
+
     
-    func activateConfirmButton() {
-        self.btnConfirm.backgroundColor = UIColor.appColors.decredGreen
-        self.lblEnterAllSeeds.isHidden = true
-        
-        // increase top spacing since warning label is now hidden so as to position button in center
-        self.tableViewFooterTopSpacingConstraint.constant = 30
-        UIView.animate(withDuration: 0.5) {
-            self.tableViewFooter.layoutIfNeeded()
-        }
+    @IBAction func backButtonTap(_ sender: Any) {
+        self.navigationController?.popViewController(animated: true)
     }
-    
-    func deactivateConfirmButton() {
-        self.btnConfirm.backgroundColor = UIColor.appColors.lightGray
+    func displayErrorLabel(){
```

Comment:
- This function should appear after the `confirm` function below, this function is called by that function.
- There should be a space before the opening brace.
- This function is not yet as intended, it should take a parameter of the error text to display, and also perform the animation (`UIView.animate`):
```
This should not always be done when the user taps confirm button, but only if there's an error to display. This can go into a function displaySeedError(_ errorMessage: String). Set the error label text there and do this animation. Of course, you'd only call that function if there's an error to display.
```